.meta/transforms/merge.toml

```diff
@@ -0,0 +1,166 @@
+[transforms.merge]
+title = "Merge"
+allow_you_to_description = "merge partial log events into a single event"
+common = false
+function_category = "merge"
+guides = []
+input_types = ["log"]
+output_types = ["log"]
+resources = []
+
+[transforms.merge.options.merge_fields]
+type = "[string]"
+common = true
+default = ["message"]
+description = """\
+Fields to merge.
+
+The values of these fields will be merged into the _first_ partial event. \
+Fields not specified here will be ignored. Merging process takes the first \
+buffered partial event, then loops over the rest of them and merges in the \
+fields from each buffered partial event. The last event with the field takes \
+priority and overrides the previous event.
```

Comment:
This logic has changed:
https://github.com/timberio/vector/blob/56571ab62595cbf700098da1a4ab8b80ab58c44b/src/transforms/merge.rs#L16-L31